src/lib/plugin_apis/swap.api

```diff
@@ -42,6 +42,7 @@ gboolean bd_swap_is_tech_avail (BDSwapTech tech, guint64 mode, GError **error);
  * bd_swap_mkswap:
  * @device: a device to create swap space on
  * @label: (nullable): a label for the swap space device
+ * @uuid: (nullable): UUID for the swap space device
```

Comment:
Looking at the `mkswap` manpage, three special values are accepted: `clear`, `random` and `time`. Does it make sense to document them here or just put a reference to the manpage? The UUID string often bears some specific format, but the usual `c1b9d5a2-f162-11cf-9ece-0020afc76f16` is being accepted here.

Reply:
I swa that too, I don't think we need to list it here, I don't really see the use case for the special options in `mkswap` (`random` is default and `clear` doesn't really make sense when creating a new swap). `swaplabel` doesn't support these.

Reply:
Fair enough, little chance of using these anyway I suppose.